src/validateEmail.test.js

```diff
@@ -8,12 +8,30 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
+    expect(typeof validateEmail('')).toBe('boolean');
+  });
+
+  it(`should return 'true' for the valid email`, () => {
+    expect(validateEmail('hello96@ukr.net')).toBe(true);
+  });
 
+  it(`should return 'false' if email starts with dot`, () => {
+    expect(validateEmail('.hello96@ukr.net')).toBe(false);
   });
 
-  it(`should return 'true' for the valid email...`, () => {
+  it(`should return 'false' if level domain starts with dot`, () => {
+    expect(validateEmail('hello96@.ukr.net')).toBe(false);
+  });
 
+  it(`should return 'true' if email contain character`, () => {
+    expect(validateEmail('hel-lo96@ukr.net')).toBe(true);
   });
 
-  // write more tests here
+  it(`should return 'false' if email doesn't contain @`, () => {
+    expect(validateEmail('hello96ukr.net')).toBe(false);
+  });
+
+  it(`should return 'true' if email contain digits`, () => {
```

Comment:
this is an unnecessary test case, you test such a condition in the first test case